tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-blacklist-test-errors-regexes.json

```diff
@@ -1,11 +1,8 @@
 {
 	"jts": [
 		".*Invalid content was found starting with element 'iiop:ejb-name'.*",
-		".*Referenced file contains errors.*jboss-ejb3-spec-2_0\\.xsd.*"
-	],
-	"ejb-security-interceptors": [
-		".*Invalid content was found starting with element 'jee:interceptor-binding'.*",
-		".*Referenced file contains errors.*jboss-ejb3-spec-2_0\\.xsd.*"
+		".*Referenced file contains errors.*jboss-ejb3-spec-2_0\\.xsd.*",
+		".*cvc-complex-type.2.4.a: Invalid content was found starting with element 'iiop:binding-name'. One of '.\"urn:iiop\":ejb-name.' is expected.*"
```

Comment:
Row 3 and 5 are quite similar, do you need them both?

Reply:
Replaced by 1 line.